src/w3d/math/tri.h

```diff
@@ -97,7 +97,7 @@ inline bool Point_In_Triangle_2D(const Vector3 &tri_point0,
     } else {
         float p0p1dist2 = p0p1.Length2();
         float p1p2dist2 = p1p2.Length2();
-        float p2p0dist2 = p1p2.Length2();
+        float p2p0dist2 = p2p0.Length2();
```

Comment:
This is also addressed in
* #319

Why we do not merge it is not entirely clear, probably because it is original game bug.

Reply:
yeah im not accepting bugfixes that we don't know the consequences of, for all we know the maps may be layed out to avoid this issue they couldn't find